platform/darwin/src/MGLStyleValue.h

```diff
@@ -2,9 +2,27 @@
 #import <CoreGraphics/CoreGraphics.h>
 
 #import "MGLFoundation.h"
+#import "MGLTypes.h"
 
 NS_ASSUME_NONNULL_BEGIN
 
+// TODO: API docs
+typedef NSString *MGLStyleFunctionOption NS_STRING_ENUM;
+
+// TODO: API docs
+extern MGL_EXPORT const MGLStyleFunctionOption MGLStyleFunctionOptionInterpolationBase;
+
+// TODO: API docs
+extern MGL_EXPORT const MGLStyleFunctionOption MGLStyleFunctionOptionDefaultValue;
+
+// TODO: API docs
+typedef NS_ENUM(NSUInteger, MGLStyleFunctionStopType) {
```

Comment:
I like that and thanks for referencing the helpful SpriteKit example. My only concern is that, with DDS, the identify function actually uses the notion of stops only as a placeholder and interpolation is not a factor. I suppose that, no matter what we call this enum, some clarification about what an identity function is will be required.